react/features/welcome/components/RootNavigationContainer.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import React from 'react';
+import { SafeAreaProvider } from 'react-native-safe-area-context';
+import { useSelector } from 'react-redux';
+
+import {
+    dialInSummaryScreenOptions,
+    drawerNavigatorScreenOptions,
+    navigationContainerTheme
+}
+    from '../../conference/components/native/ConferenceNavigatorScreenOptions';
+import { screen } from '../../conference/components/native/routes';
+import { DialInSummary } from '../../invite';
+import { isWelcomePageAppEnabled } from '../functions';
+
+import BlankPage from './BlankPage';
+import { rootNavigationRef } from './RootNavigationContainerRef';
+
+import { WelcomePageNavigationContainer } from './';
+
+const RootStack = createStackNavigator();
+
+
+const RootNavigationContainer = () => {
+    const isWelcomePageAvailable = useSelector(isWelcomePageAppEnabled);
+
+    return (
+        <SafeAreaProvider>
+            <NavigationContainer
+                independent = { true }
+                ref = { rootNavigationRef }
+                theme = { navigationContainerTheme }>
+                <RootStack.Navigator
+                    initialRouteName = { screen.welcome.main }>
```

Comment:
IS this correct, when the welcome page is disabled?

Reply:
Based on their documentation, it should be.
https://reactnavigation.org/docs/5.x/auth-flow